tests/test_integration/test_hook_implementations.py

```diff
@@ -0,0 +1,53 @@
+import fmu.dataio.hook_implementations.jobs
+from ert.shared.plugins.plugin_manager import ErtPluginManager
+from fmu.dataio.scripts import create_case_metadata
+
+
+def test_hook_implementations():
+    plugin_manager = ErtPluginManager(
+        plugins=[
+            fmu.dataio.hook_implementations.jobs,
+            create_case_metadata,
+        ]
+    )
+
+    installable_fms = plugin_manager.get_installable_jobs()
+    expected_forward_models = {}
+    for fm_name, fm_location in expected_forward_models.items():
+        assert fm_name in installable_fms
+        assert installable_fms[fm_name].endswith(fm_location)
```

Comment:
This does not look right, the loop will always be "skipped".

Reply:
Yep, thanks, fix'd it